bin/build-docker-de

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -xue
+
+cleanup() {
+  rm -f $CHAIN/docker/de/cored
+  rm -f $CHAIN/docker/de/corectl
+  rm -f $CHAIN/docker/de/build-cored-release
+}
+trap cleanup EXIT
+
+imageVersion=${IMAGE_VERSION:-1.0.0}
+coredVersion=${CORED_VERSION:-cmd.cored-1.0.1}
+
+cp bin/build-cored-release $CHAIN/docker/de
```

Comment:
Is there a reason to copy this script into the target directory? It should be runnable from anywhere.